features/config-is-true.feature

```diff
@@ -0,0 +1,57 @@
+Feature: Determine whether the value of a constant or variable defined in wp-config.php is true.
+  Background:
+    Given an empty directory
+    And a wp-includes/version.php file:
+   """
+   <?php
+   $wp_version = '6.3';
+   """
+    And wp-config.php
+
+  Scenario Outline: Get the value of a variable whose value is true
+    When I run `wp config set <variable> <value> --type=<type> <raw>`
```

Comment:
Instead of using `wp config set` to write the value, what do you think about using `And a wp-config.php file:` instead?

```
And a `wp-config.php` file:
  """
  <?php
  define( 'WP_TRUTH', true );
  """
```

I think my suggested approach has a couple of advantages:
1. We avoid an (unlikely) scenario where `wp config set` is buggy, and then `wp config is-true` is buggy too.
2. The tests will make it much more obvious how a given value in `wp-config.php` correlates with its `wp config is-true` value.

What do you think?

Reply:
Figured out what was going wrong with the behat tests and updated them to use the new method as well as adding tests to ensure that it works correctly with PHP include files.